.travis.yml

```diff
@@ -93,7 +93,7 @@ git:
 before_script:
   - export PLUGIN_SLUG=$(basename $(pwd))
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
-  - composer install
+  - composer install --ignore-platform-reqs
```

Comment:
Added this to a number of `composer install` calls because the phpcs-changed does not work on PHP 5.6. This tells composer to ignore the local version of PHP requested and just push forward with the install.

I'm unsure how I feel about it. Since we are only using our own packages in production, this only impacts development work. What do others think if this is a fair tradeoff?